src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Ball {
+    private final Color color;
+    private final int number;
+
+    protected Ball(Color color, int number) {
```

Comment:
why protected? the usage of protected is very limited in practice, let's better be not afraid to use public

Reply:
are you talking about this place specificly or about access modifiers in whole project?

Reply:
in whole project